metrics/web/lib/auth/presentation/models/auth_error_message.dart

```diff
@@ -27,6 +27,8 @@ class AuthErrorMessage {
         return AuthStrings.unknownErrorMessage;
       case AuthErrorCode.googleSignInError:
         return AuthStrings.googleSignInErrorOccurred;
+      case AuthErrorCode.notAllowedEmailDomain:
```

Comment:
Could you please add more tests to the auth error message tests to cover all available mapping cases?